### ISSUE TITLE
Using Git

### ISSUE THREAD

**Issue by chinnichaitanya** [ShaastraWebops/Frontend-2016#8]
Here is essentially what you have to do after you make changes to local repo and want to push:
1. `git add --all`
2. `git commit -a -m "Message to use in commit. It should explain changes youve made"`
3. `git pull origin master`
4. `git push origin master`

If you get any conflicts, It will mention which files have a conflict, you go to that file and open it. Somewhere, you'll see something like :
```
<<<<<<<<<< HEAD
_some code which you wrote here_
---------------------------
_some code which others may have written here_
>>>>>>>>>>> 868348347r847r34r6t384r3
```
You need to remove the `<<<<<<<<<< HEAD` and the `----------------` and the `>>>>>>>>>>> 868348347r847r34r6t384r3` and choose which ever you want to keep. The old code or the new code. If you want some files from old code and some from new, you can delete only specific lines in both.
Basically, what you put here will be exactly how the file will get stored, so be careful not to delete anything important, be careful not to add wrong code.

**Comment by chinnichaitanya** [comment]
I'm telling this again, don't mess up the order of the git commands while making the commit and don't mess up the repo.
